api/services/ChatService.js

```diff
@@ -13,11 +13,7 @@ function getChats() {
 }
 
 function addUserMessage(chat) {
-  if (!chat.username) {
-    chat.username = 'Anonymous';
-  }
-  chat.type = 'user';
-  chat.time = new Date(chat.time).toISOString()
+  chat.time = new Date().toISOString();
```

Comment:
this won't represent the time the user submitted it, it'll be the time when the server gets to processing it